widgets/GoogleSheets-batchsource.json

```diff
@@ -0,0 +1,503 @@
+              "value": "false",
+              "label": "No"
+            },
+            "default": "false"
+          }
+        },
+        {
+          "widget-type": "toggle",
+          "label": "Add spreadSheet/sheet name fields",
+          "name": "addNameFields",
+          "widget-attributes": {
+            "on": {
+              "value": "true",
+              "label": "Yes"
+            },
+            "off": {
+              "value": "false",
+              "label": "No"
+            },
+            "default": "false"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "SpreadSheet field name",
+          "name": "spreadSheetFieldName",
+          "widget-attributes": {
+            "default": "spreadSheetName"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Sheet field name",
+          "name": "sheetFieldName",
+          "widget-attributes": {
+            "default": "sheetName"
+          }
+        },
+        {
+          "widget-type": "radio-group",
+          "label": "Column Names Selection",
+          "name": "columnNamesSelection",
+          "widget-attributes": {
+            "layout": "inline",
+            "default": "firstRowAsColumns",
+            "options": [
+              {
+                "id": "firstRowAsColumns",
+                "label": "Treat first row as column names"
+              },
+              {
+                "id": "customRowAsColumns",
+                "label": "Custom row as column names"
+              },
+              {
+                "id": "noColumnNames",
+                "label": "No column names"
+              }
+            ]
+          }
+        },
+        {
+          "widget-type": "number",
+          "label": "Custom Row For Column Names",
+          "name": "customColumnNamesRow",
+          "widget-attributes": {
+            "default": "1",
+            "min": "1"
+          }
+        },
+        {
+          "widget-type": "number",
+          "label": "Last Data Column",
+          "name": "lastDataColumn",
+          "widget-attributes": {
+            "default": "26",
+            "min": "0"
+          }
+        },
+        {
+          "widget-type": "number",
+          "label": "Last Data Row",
+          "name": "lastDataRow",
+          "widget-attributes": {
+            "default": "1000",
+            "min": "0"
+          }
+        },
+        {
+          "widget-type": "number",
+          "label": "Read Buffer Size",
+          "name": "readBufferSize",
+          "widget-attributes": {
+            "default": "100",
+            "min": "1"
+          }
+        }
+      ]
+    }
+  ],
+  "outputs": [
+    {
+      "widget-type": "non-editable-schema-editor",
+      "schema": {
+      }
+    }
+  ],
+  "filters": [
+    {
+      "name": "Select modification date range",
+      "condition": {
+        "expression": "modificationDateRange == 'custom'"
```

Comment:
if possible we should always use "equalTo" or other predefined checks.
https://github.com/data-integrations/http/blob/06fca99995efac5f7f15147b5489dd3c493316a3/widgets/HTTP-batchsource.json#L523

Reply:
done